### PR TITLE
LLM, Translation Tokens size added in telemetry.

### PR DESCRIPTION
**Pull request by piyushbhavsar12**
1. LLM_TYPE
2. TRANSLATION_TYPE 
3. BEDROCK_MODEL_ID
4. input token size
5. output token size
6. total token size

### CODE REVIEW

File: telemetry_logger.py [hunk 1]
```diff
@@ -5,10 +5,15 @@
...
 from logger import logger
+from dotenv import load_dotenv
+load_dotenv()
 
 telemetryURL = os.getenv('TELEMETRY_ENDPOINT_URL')
 ENV_NAME = os.getenv('SERVICE_ENVIRONMENT')
 TELEMETRY_LOG_ENABLED = get_from_env_or_config('telemetry', 'telemetry_log_enabled', None).lower() == "true"
+LLM_TYPE=os.getenv("LLM_TYPE")
+TRANSLATION_TYPE=os.getenv("TRANSLATION_TYPE")
+BEDROCK_MODEL_ID=os.getenv("BEDROCK_MODEL_ID")
```
Comment (ankush-maherwal) [inline telemetry_logger.py:16]: @piyushbhavsar12 We don't need to log debrock_model_id. This will not be used if we use openai model

Comment (piyushbhavsar12) [reply telemetry_logger.py:16]: Okay, I have removed it. Could you please check again now?

Comment (ankush-maherwal) [reply telemetry_logger.py:16]: @piyushbhavsar12 Merging the code for now, but we need to have a common way to calculate token size for all LLM

Comment (manojLondhe) [reply telemetry_logger.py:16]: BEDROCK_MODEL_ID remove from here too @piyushbhavsar12

**Pull request merged**